docs/reference/mapping/types/date.asciidoc

```diff
@@ -3,7 +3,7 @@
 
 JSON doesn't have a date datatype, so dates in Elasticsearch can either be:
 
-* strings containing formatted dates, e.g. `"2015-01-01"` or `"2015/01/01 12:10:30"`.
+* strings containing formatted dates, e.g. `"2015-01-01"` or `"2015-01-01T12:10:30"`.
```

Comment:
Interestingly dynamic mapping detects the correct format. If the first date indexed is something like `"2015/01/01 12:10:30"` if maps it to a date field with format "yyyy/MM/dd HH:mm:ss||yyyy/MM/dd||epoch_millis", at least in 6.5 where I just checked. But I agree it might be confusing if you already have a date field mapped with the default format.